app/src/main/java/io/github/gsantner/memetastic/ui/MemeItemAdapter.java

```diff
@@ -166,6 +251,28 @@ private void toggleFavorite(ViewHolder holder) {
         }
     }
 
+    private void toggleHidden(ViewHolder holder, int position){
+        MemeData.Image image = (MemeData.Image) holder.imageView.getTag();
+        String filePath = image.fullPath.getAbsolutePath();
+
+        if (_app.settings.toggleHiddenMeme(filePath)){
```

Comment:
are you sure that both toggle cases should dl the same?

Reply:
Yes, if meme is being hidden it is removed from the create tab and added to the hidden tab and if the action is occurring in the hidden tab the meme is removed from the hidden tab and added to the create tab

Reply:
but why is there an if-condition where both do the exactly same?

Reply:
Lol, can't see how i missed that, would fix it.

It was initially toggling the template status of a meme but you said hidden shouldn't affect a meme being a template

Reply:
ahh